src/main/java/net/homework/webapp/service/EmployeeService.java

```diff
@@ -0,0 +1,15 @@
+package net.homework.webapp.service;
+
+import net.homework.webapp.model.Employee;
+import java.util.List;
+
+public interface EmployeeService {
+
+    List<Employee> getAllEmployees();
+
+    Employee getById(final Long id);
+
+    Employee update(final Long id, Employee employee, final Long departmentId);
```

Comment:
can we get departmentId from employee?

Reply:
did not fixed because we can't get departmentId from employee